src/main/java/com/epam/training/auction/services/AuctionsServiceImpl.java

```diff
@@ -6,52 +6,59 @@
-    private static final Logger LOGGER = Logger.getLogger(AuctionsServiceImpl.class);
-    private static final String AUCTION_NOT_FOUND_MESSAGE = "Auction with id %d was not found";
+  private static final Logger LOGGER =
+      Logger.getLogger(AuctionsServiceImpl.class);
+  private static final String AUCTION_NOT_FOUND_MESSAGE =
+      "Auction with id %d was not found";
 
-    private AuctionRepository auctionRepository;
+  @Autowired
+  private UserRepository userRepository;
 
-    @Autowired
-    public AuctionsServiceImpl(AuctionRepository auctionRepository) {
-        this.auctionRepository = auctionRepository;
-    }
+  private AuctionRepository auctionRepository;
 
-    public List<AuctionTransferObject> getActiveAuctions() {
-        List<Auction> auctions = auctionRepository.findByActiveTrue();
-        return AuctionMapper.map(auctions);
-    }
+  @Autowired
+  public AuctionsServiceImpl(AuctionRepository auctionRepository) {
+    this.auctionRepository = auctionRepository;
+  }
 
-    public List<AuctionTransferObject> getArchivedAuctions() {
-        List<Auction> auctions = auctionRepository.findByActiveFalse();
-        return AuctionMapper.map(auctions);
-    }
+  public List<AuctionTransferObject> getActiveAuctions() {
+    List<Auction> auctions = auctionRepository.findByActiveTrue();
+    return AuctionMapper.map(auctions);
+  }
 
-    public Long addAuction(AuctionTransferObject auctionTransferObject) {
-        User seller = new User(auctionTransferObject.getSeller().getUsername(), auctionTransferObject.getSeller().getPassword());
-        Auction auction = new Auction(auctionTransferObject.getTitle(), auctionTransferObject.getDescription(), seller);
-        auction = auctionRepository.save(auction);
-        return auction.getId();
-    }
+  public List<AuctionTransferObject> getArchivedAuctions() {
+    List<Auction> auctions = auctionRepository.findByActiveFalse();
+    return AuctionMapper.map(auctions);
+  }
 
-    public AuctionTransferObject getAuctionById(long id) {
-        Auction auction = auctionRepository.findOne(id);
+  public Long addAuction(AuctionTransferObject auctionTransferObject) {
+    User seller = userRepository.findByUserName(auctionTransferObject.getSeller().getUsername());
+    Auction auction = new Auction(auctionTransferObject.getTitle(),
+        auctionTransferObject.getDescription(), seller);
+    auction.setActive(true);
+    auction = auctionRepository.save(auction);
+    return auction.getId();
+  }
 
-        if (auction == null) {
-            throw new ItemNotFoundException(format(AUCTION_NOT_FOUND_MESSAGE, id));
-        }
+  public AuctionTransferObject getAuctionById(long id) {
+    Auction auction = auctionRepository.findOne(id);
 
-        return AuctionMapper.map(auction);
+    if (auction == null) {
+      throw new ItemNotFoundException(format(AUCTION_NOT_FOUND_MESSAGE, id));
     }
+
+    return AuctionMapper.map(auction);
+  }
 }
```

Comment:
please change formatting in the IDE to match with rest of team. 4 symbols instead of 2